sdk/test/trace/tracer_test.cc

```diff
@@ -223,18 +226,105 @@ TEST(Tracer, GetSampler)
   ASSERT_EQ("AlwaysOffSampler", t2->GetDescription());
 }
 
-TEST(Tracer, SpanSetAttribute)
+TEST(Tracer, ComprehensiveTest)
```

Comment:
nit: ComprehensiveTest doesn't actually have much semantic meaning. Consider something closer to what you're actually testing, ie. TraceSamplingTest.

Reply:
Changed to Test___Sampler